.gitignore

```diff
@@ -55,3 +55,4 @@ docs/_build/
 
 *.so
 /venv
+.eggs
```

Comment:
@honzajavorek I can't help but to comment on this. What's eggs? Is this supposed to be an easter egg?